landoapi/transactions.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Functions for working with Phabricator transactions."""
+from landoapi.phabricator import PhabricatorClient
+
+
+def transaction_search(phabricator, object_identifier, limit=100):
```

Comment:
We'll want a way to specify the constraints. At least the phid one. Maybe that's coming in a future change though when you make use of this function?

Reply:
The missing PHID constraint is a hold-over from the previous implementation of revision comment retrieval.  I've added a new parameter to the method to constrain searches to specific PHIDs.